app/Eloquent/UpdateBook.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+
+class UpdateBook extends Model
+{
+    protected $table = 'update_books';
+
+    protected $fillable = [
+    	'title',
+    	'description',
+    	'author'
+    	'pulish_date',
+    	'total_pages',
+    	'sku',
+    	'user_id',
+    	'book_id',
+    ];
+
+    public function user()
+    {
+    	return $this->belongsTo(User::class);
+    }
+
+    public function book()
+    {
+    	return $this->belongsTo(Book::class);
+    }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Eloquent/UpdateBook.php</br>Spaces must be used to indent lines; tabs are not allowed